src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/UntypedBlankAsTable.txt

```diff
@@ -1,42 +1,47 @@
-#SETUP: ConsistentOneColumnTableResult
 
-// Using an untyped blank, such as the return type from Blank(), as an argument to a single column table enabled function.
-// We are inconsistent, but don't want to break existing behavior, and this scenario is not common or useful.
+// Consider an untyped blank, such as the return type from Blank(), as an argument to a single column table enabled function.
+//
+// We favor treating these as a scalar blank, rather than a table blank.  Scalar is the simpler form and more likely
+// what the maker wanted.  This treatment is also consistent with Canvas.
+//
 // A typed Blank, such as returned from a database or If(1<0,"foo") is fine, it specifically an untyped Blank that is odd.
+//
 // In general, a function that takes a blank table argument will return a blank table.
 
 >> Concatenate("a",Blank(),"b")
 "ab"
 
 >> Concatenate(["a","b"],Blank())
-Blank()
+Table({Value:"a"},{Value:"b"})
 
 >> Concatenate(Blank(),["a","b"])
-Blank()
+Table({Value:"a"},{Value:"b"})
 
 >> Left(["abc","def"],2)
 Table({Value:"ab"},{Value:"de"})
 
 >> Left(["abc","def"],Blank())
-#SKIP: Waiting on https://github.com/microsoft/Power-Fx/issues/1317, JS and C# disagree
+Table({Value:""},{Value:""})
 
 >> Left("abcd",[2,3])
 Table({Value:"ab"},{Value:"abc"})
 
+// suspect, may be different on Canvas requiring an override
 >> Left(Blank(),[2,3])
-#SKIP: Waiting on https://github.com/microsoft/Power-Fx/issues/1317, JS and C# disagree
+Table({Value:""},{Value:""})
 
 >> Right(["abc","def"],2)
 Table({Value:"bc"},{Value:"ef"})
 
 >> Right(["abc","def"],Blank())
-#SKIP: Waiting on https://github.com/microsoft/Power-Fx/issues/1317, JS and C# disagree
+Table({Value:""},{Value:""})
 
 >> Right("abc",[2,3])
 Table({Value:"bc"},{Value:"abc"})
 
+// suspect, may be different on Canvas requiring an override

```

Comment:
There are three tests that give different results in Canvas.  We need to validate that these are the right answers and that Canvas should snap to these semantics with Power Fx 1.0.   @CarlosFigueiraMSFT

Reply:
This works as expected in Power Apps

Reply:
My fault, I neglected to turn on Excel Compat.  The test is OK.